test/integration/targets/openssl_csr/tasks/CentOS.yml

```diff
@@ -0,0 +1,18 @@
+- name: Install epel
+  become: yes
+  package:
+    name: epel-release
+    state: latest
+
+- name: Install python-pip
+  become: yes
+  package:
+    name: python-pip
+    state: latest
```

Comment:
You shouldn't need to install pip. That is handled by `ansible-test` as part of the test infrastructure.

The same is true for the other tests.